cmd/main.go

```diff
@@ -142,6 +173,46 @@ Full docs can be found at github.com/aws/amazon-` + binName
 	}
 	resultsOutputFn := outputs.SimpleInstanceTypeOutput
 
+	cliSortDirections := []string{
+		sortAscending,
+		sortAsc,
+		sortDescending,
+		sortDesc,
+	}
+
+	sortingShorthandFlags := []string{
+		vcpus,
+		memory,
+		gpuMemoryTotal,
+		networkInterfaces,
+		spotPrice,
+		odPrice,
+		instanceStorage,
+		ebsOptimizedBaselineBandwidth,
+		ebsOptimizedBaselineThroughput,
+		ebsOptimizedBaselineIOPS,
+		gpus,
+		inferenceAccelerators,
+	}
+
+	sortingShorthandPaths := []string{
+		vcpuPath,
+		memoryPath,
+		gpuMemoryTotalPath,
+		networkInterfacesPath,
+		spotPricePath,
+		odPricePath,
+		instanceStoragePath,
+		ebsOptimizedBaselineBandwidthPath,
+		ebsOptimizedBaselineThroughputPath,
+		ebsOptimizedBaselineIOPSPath,
+		gpus,
+		inferenceAccelerators,
+	}
```

Comment:
I think you can simplify this and just use a map. Then you can get rid of the mapping func below.

```suggestion
	sortingShorthands := map[string]string{
		vcpus: vcpuPath,
		memory: memoryPath,
		gpuMemoryTotal: gpuMemoryTotalPath,
		networkInterfaces: networkInterfacesPath,
		spotPrice: spotPricePath,
		odPrice: odPricePath,
		instanceStorage: instanceStoragePath,
		ebsOptimizedBaselineBandwidth: ebsOptimizedBaselineBandwidthPath,
		ebsOptimizedBaselineThroughput: ebsOptimizedBaselineThroughputPath,
		ebsOptimizedBaselineIOPS: ebsOptimizedBaselineIOPSPath,
		gpus: gpus,
		inferenceAccelerators: inferenceAccelerators,
	}
```

Reply:
That is a very good catch. I initially had the arrays since I used the `sortingShorthandFlags` array to display the available flags in the `--help` printout, but since we moved that to the README, there is no need to have the arrays before the creation of the map.